modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -1198,13 +1198,19 @@ Maximum number of bytes returned in a fetch request.
 
 The strategy used to fulfill fetch requests.
 
+* `polling`: Repeatedly polls every partition in the request for new data. The polling interval is set by <<fetch_reads_debounce_timeout,`fetch_reads_debounce_timeout`>>.
+
+* `non_polling`: The backend is signaled when a partition has new data, so it doesn’t need to repeatedly read from every partition in the fetch. This should improve fetch latency and CPU utilization for most workloads.
+
+* `non_polling_with_debounce`: This option behaves like `non_polling` but includes a debounce mechanism with a fixed delay specified by <<fetch_reads_debounce_timeout,fetch_reads_debounce_timeout>> at the start of each fetch. This delay results in fewer, larger fetch responses, reducing overall reactor utilization and allowing other requests, such as produces, to increase. 
```

Comment:
What does 'allowing produce requests to increase' mean?     I think we mean it frees up resources for other requests. Consider rewording

Reply:
done